apps/wakunode2/app.nim

```diff
@@ -44,8 +43,40 @@ import
 when defined(rln):
   import ../../waku/v2/protocol/waku_rln_relay
 
+logScope:
+  topics = "wakunode app"
 
-type AppResult[T] = Result[T, string]
+
+# Git version in git describe format (defined at compile time)
+const git_version* {.strdefine.} = "n/a"
+
+type
+  App* = object
+    version: string
+    conf: WakuNodeConf
+
+    rng: ref HmacDrbgContext
+    peerStore: Option[WakuPeerStorage]
```

Comment:
Should we have a getter for peerStorage for tests? (considering that we create the peerStore in setupPeerPersistence)

Reply:
For now, I am reluctant to expose the internal implementation details for testing purposes. Once we move to the `App` type altogether, we can start refactoring things.